app/controllers/tasklist_controller.rb

```diff
@@ -0,0 +1,80 @@
+class TasklistController < ApplicationController
+  def index
+      @all_tasklist = RailsTaskList.all
+  end
+
+
+    # def by_title
+    #   @tasklist = RailsTaskList.where(title: params[:title])
+    # end
+
+    def new
+      @tasklist = RailsTaskList.new
+      @all_people = Person.all
+    end
+
+    def finished
+      @tasklist = RailsTaskList.find(params[:id])
+      @tasklist.update(completion_status: "Done!")
+      redirect_to root_path
+    end
+
+    def edit
+      @tasklist = RailsTaskList.find(params[:id])
+      @all_people = Person.all
+
+      render :new
+    end
+
+    def update
+      @tasklist = RailsTaskList.find(params[:id])
+      if @tasklist.update(edit_params[:rails_task_list])
+        redirect_to tasklist_path(@tasklist.id)
+      else
+
+        render :new
+      end
+    end
+
+
+
+    def show
+      @all_tasklist = RailsTaskList.find(params[:id])
+    end
+
+    def create
+      @tasklist = RailsTaskList.new( tasklist_params[:rails_task_list])
+
+      if @tasklist.save
+        redirect_to root_path
+      else
+      render :new
+      end
+    end
+
+    def delete
+
+        @tasklist = RailsTaskList.find(params[:id])
```

Comment:
You repeat this line in a few places, so you could possibly it out into a `before_action` only for certain controller methods if you don't think that would reduce clarity.